### PR TITLE
update validators in transactions/multisig panel

### PR DESCRIPTION
**Pull request by NemTechCN**
1. create ErrorMessage component, this is a component to show error message in from;
2. create validator functions in infrastructure.

### CODE REVIEW

File: src/components/multisig/MultisigConversion.vue [hunk 1]
```diff
@@ -233,15 +212,17 @@ import {
   UInt64,
 } from 'nem2-sdk';
 import { mapState } from 'vuex';
-import ErrorMessage from '../../infrastructure/transactions/errorMessage';
+import ErrorMessage from '../errorMessage/ErrorMessage.vue';
 import Confirmation from '../signature/Confirmation.vue';
 import SendConfirmation from '../signature/SendConfirmation.vue';
+import { multisigCovertionValidator } from '../../infrastructure/multisig/multisigFormValidator';
```
Comment (evias) [inline src/components/multisig/MultisigConversion.vue]: `multisigConversion` - typo ?

File: src/infrastructure/multisig/multisigFormValidator.js [hunk 2]
```diff
@@ -0,0 +1,96 @@
+import ErrorMessage from '../errorMessage/error-message';
+
+export const multisigModifyValidator = (pointer) => {
```
Comment (evias) [inline src/infrastructure/multisig/multisigFormValidator.js]: not sure I understand this "validator" alternative. Vue probably provides with a Validator interface no ? 

@decentraliser please have a look here too.

File: src/infrastructure/multisig/multisigFormValidator.js [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+    return errorMessage;
+  }
+  if (cosignerResult > 10) {
+    errorMessage.push(ErrorMessage.TOO_MANY_COSIGNERS);
+    return errorMessage;
+  }
+  return errorMessage;
+};
+
+export const multisigCovertionValidator = (pointer) => {
```
Comment (evias) [inline src/infrastructure/multisig/multisigFormValidator.js]: rename to `multisigConversionValidator`

File: src/components/multisig/MultisigConversion.vue (hunk 1 above)
Comment (NemTechCN) [reply src/components/multisig/MultisigConversion.vue]: fixed

File: src/infrastructure/multisig/multisigFormValidator.js (hunk 2 above)
Comment (NemTechCN) [reply src/infrastructure/multisig/multisigFormValidator.js]: fixed

File: src/infrastructure/multisig/multisigFormValidator.js (hunk 3 above)
Comment (NemTechCN) [reply src/infrastructure/multisig/multisigFormValidator.js]: fixed

**Pull request closed without merging**